src/Shape/Whisker.js

```diff
@@ -0,0 +1,90 @@
+
+import {select} from "d3-selection";
+
+import {assign, BaseClass, constant, elem} from "d3plus-common";
+
+import Path from "./Path";
+
+/**
+    @class Whisker
+    @extends BaseClass
+    @desc Creates SVG lines based on an array of data.
+*/
+export default class Whisker extends BaseClass {
+
+  /**
+      @memberof Whisker
+      @desc Invoked when creating a new class instance, and overrides any default parameters inherited from BaseClass.
+      @private
+  */
+  constructor() {
+
+    super();
+
+    this._name = "Whisker";
+    this._pathConfig = {
+      stroke: constant("black"),
+      strokeWidth: constant(1)
+    };
+
+  }
+
+  /**
+      @memberof Whisker
+      @desc Draws the whisker.
+      @chainable
+  */
+  render() {
+
+    if (this._select === void 0) {
+      this.select(select("body").append("svg")
+        .style("width", `${window.innerWidth}px`)
+        .style("height", `${window.innerHeight}px`)
+        .style("display", "block").node());
+    }
+
+    new Path()
+      .data(this._data)
+      .select(elem("g.d3plus-Whisker", {
+        parent: this._select
+      }).node())
+      .strokeWidth(1)
+      .render();
+
+    console.log("this._data: ", this._data);
+    return this;
+  }
+
+  /**
+      @memberof Whisker
+      @desc If *data* is specified, sets the data array to the specified array and returns the current class instance. If *data* is not specified, returns the current data array.
+      @param {Array} [*data* = []]
+      @chainable
+  */
+  data(_) {
+    return arguments.length
+      ? (this._data = _, this)
+      : this._data;
+  }
+
+  /**
+      @memberof Whisker
```

Comment:
add a `length` method, which represents the pixel value of the line connecting the whisker's endpoint to it's origin